Replicator/ChangesFeed.cc

```diff
@@ -262,7 +261,14 @@ namespace litecore::repl {
     ReplicatorChangesFeed::ReplicatorChangesFeed(Delegate& delegate, const Options* options, DBAccess& db,
                                                  Checkpointer* cp)
         : ChangesFeed(delegate, options, db, cp)  // DBAccess is a subclass of access_lock<C4Database*>
-        , _usingVersionVectors(db.usingVersionVectors()) {}
+        , _usingVersionVectors(db.usingVersionVectors()) {
+        if ( _options->push(_collectionIndex) == kC4Continuous ) _db.echoCanceler.trackCollection(_collectionIndex);
+    }
+
```

Comment:
"only for collections that have bidirectional continuous replication" - but looks like either above, (continuous push), or below (continuous pull), would trigger trackCollection?

Reply:
+1